.github/workflows/ci.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         uses: FreeRTOS/CI-CD-Github-Actions/doxygen@main
         with:
           path: ./
+          doxygen_link: https://sourceforge.net/projects/doxygen/files/rel-1.9.4/doxygen-1.9.4.linux.bin.tar.gz
```

Comment:
We should move to 1.9.5. This is the version we're using in every upgraded package and is the latest doxygen.